backend/Dtos/Account/RegisterDto.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Runtime.Intrinsics.X86;
+using System.Threading.Tasks;
+
+namespace backend.Dtos.Account
+{
+    public class RegisterDto
+    {
+        [Required]
+        public string? UserName { get; set; }
+        [Required]
+        [EmailAddress]
+        public string? Email { get; set; }
+        [Required]
+        public string? Password { get; set; }
+        [Required]
+        public string? FirstName { get; set; }
+        [Required]
+        public string? LastName { get; set; } 
+
```

Comment:
The empty line can be removed for consistency with other files

Reply:
Removed